pkg/apis/harvesterhci.io/v1beta1/settings.go

```diff
@@ -6,7 +6,8 @@ import (
 )
 
 var (
-	SettingConfigured condition.Cond = "configured"
+	SettingConfigured      condition.Cond = "configured"
+	SettingPreconfigLoaded condition.Cond = "preconfigLoaded"
```

Comment:
@gitlawr @guangbochen we have two different naming styles for condition type in the code. Some start with upper case and some start with lower case. Do we want to be consistent? Conditions in Kubernetes resources seem to use upper case more.

Reply:
We want to use a consistent naming style. For that change, we need to consider upgrades and coordination with the UI. We'd better track it in a separate issue.